apps/gitness/src/pages-v2/create-project/create-project-container.tsx

```diff
@@ -0,0 +1,46 @@
+import { useNavigate } from 'react-router-dom'
+
+import { OpenapiCreateSpaceRequest, useCreateSpaceMutation } from '@harnessio/code-service-client'
+import { CreateProjectPage } from '@harnessio/ui/views'
+
+import { useAppContext } from '../../framework/context/AppContext'
+import { useTranslationStore } from '../../i18n/stores/i18n-store'
+
+export default function CreateProject() {
+  const navigate = useNavigate()
+  const { addSpaces } = useAppContext()
+
+  // Set up the mutation hook with the form data
+  const { mutate, isLoading, error } = useCreateSpaceMutation(
+    {},
+    {
+      onSuccess: response => {
+        const { body: project } = response
+        addSpaces([project])
+        //onSuccess in react-query has allowed 200-299
+        navigate(`/${project?.identifier}/repos`)
+      }
+    }
+  )
+
+  const handleFormSubmit = (formData: OpenapiCreateSpaceRequest) => {
+    // Trigger the mutation with form data as the request body
+    mutate({
+      body: {
+        identifier: formData.identifier || '',
+        description: formData.description || '',
+        is_public: formData.is_public ?? false,
+        parent_ref: formData.parent_ref || ''
```

Comment:
why do we need empty fallbacks for all fields? i assume atleast some of them are required to call this API? that should reflect in the form and it's schema

Reply:
Turns out our form only takes in an `identifier`. Unsure why we even had the rest of the fields in the schema. Fixed.